### PR TITLE
Add theming option

### PR DESCRIPTION
**Pull request by rossdeane**
Solves #30

Support some sort of theme system with the flag `--theme`

Current options are `light`, `dark`

### CODE REVIEW

**Comment by muesli** [comment]
Nice! We should use termenv's HasDarkBackground() (https://godoc.org/github.com/muesli/termenv#HasDarkBackground) to set a reasonable default.

**Comment by rossdeane** [comment]
Clever! I'll have a look shortly 👍

**Comment by muesli** [comment]
Awesome! I also wonder if we can somehow integrate a "fix" for #49.

**Comment by rossdeane** [comment]
Now using `HasDarkBackground` to set defaults, but that doesn't actually seem to work with my WSL / Windows Terminal background. But that's a different story :)

How do you see it working re #49? A theme=basic or something? 
I think it would be better as a seperate flag altogether imo, then they could potentially be combined, i.e. basic + light, basic + dark.

File: themes.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		colorYellow:  term.Color("#FFAF00"),
+		colorGreen:   term.Color("#005F00"),
+		colorBlue:    term.Color("#000087"),
+		colorGray:    term.Color("#303030"),
+		colorMagenta: term.Color("#AF00FF"),
+		colorCyan:    term.Color("#0087FF"),
+	}
+
+	if _, ok := themes[theme]; !ok {
+		if !termenv.HasDarkBackground() {
```
Comment (muesli) [inline themes.go:41]: I think we shouldn't do that here: whatever theme was requested from `loadThemes` should be returned as expected. Instead we should have a method that returns either `dark` or `light` (depending on the value of `HasDarkBackground` and set that as the default for `themeOpt`.

**Comment by rossdeane-awin** [comment]
Also opened #54 for the other issue, or do you think we should combine it in a single option with this?

File: themes.go (hunk 1 above)
Comment (rossdeane) [reply themes.go]: Something like this maybe?

**Comment by rossdeane** [comment]
Also opened #54 for the other issue, or do you think we should combine it in a single option with this?
(sorry if i've spammed you I keep commiting on my work account :) )

**Comment by muesli** [comment]
Thank you for this great contribution!

**Pull request merged**